packages/python/plotly/plotly/express/_core.py

```diff
@@ -1064,14 +1062,14 @@ def _escape_col_name(df_input, col_name, extra):
     return col_name
 
 
-def to_unindexed_series(x):
+def to_unindexed_series(x, name=None):
```

Comment:
Originally, it was creating series without a name. I set it as None in case this function was used externally to avoid breaking compatibility.
When we create a dataframe from a dict, it's safer to have named series. Also, it would be easier to debug in line to see which series was created in order to know the column that caused the issue.

Reply:
The change might not be necessary, but i like the explicitness. Can be useful during debugging.